src/tools/utils.ts

```diff
@@ -93,3 +97,65 @@ ${snapshot}
     }],
   };
 }
+
+type StorageState = Awaited<ReturnType<playwright.BrowserContext['storageState']>>;
+
+export async function saveStorageStateFor(page: playwright.Page, alias: string): Promise<string> {
+  const filePath = await storageStatePath(alias);
+  await page.context().storageState({ path: filePath });
+  return filePath;
+}
+
+export async function loadStorageStateFor(page: playwright.Page, alias: string) {
```

Comment:
just to double check, this and the method above are unused in this PR. is that intended?

Reply:
Oh yes, that was a previous attempt.